src/ui/src/components/command-palette/providers/script/script-provider-common.tsx

```diff
@@ -36,22 +37,62 @@ export function quoteIfNeeded(input: string): string {
   return needQuoteRe.test(input) ? `"${input.replace(/"/g, '\\"')}"` : input;
 }
 
+const useStyles = makeStyles((theme: Theme) => createStyles({
+  labelBox: {
+    display: 'flex',
+    gap: theme.spacing(1),
+    flexFlow: 'row nowrap',
+  },
+  descTitle: { ...theme.typography.h2 },
+  descHint: {
+    ...theme.typography.subtitle2,
+    fontStyle: 'italic',
+  },
+  descBody: {
+    ...theme.typography.body1,
+    marginTop: 0,
+  },
+  descBodyFull: {
+    ...theme.typography.body1,
+    marginTop: theme.spacing(1),
+  },
+}), { name: 'CommandPaletteBits' });
+
 export const CompletionLabel = React.memo<{
   icon: React.ReactNode,
   input: string,
   highlights: number[],
 }>(({ icon, input, highlights }) => {
-  const outs: React.ReactNode[] = [];
-  for (let i = 0; i < input.length; i++) {
-    if (highlights.includes(i)) outs.push(<strong key={i}>{input.substring(i, i + 1)}</strong>);
-    else outs.push(<span key={i}>{input.substring(i, i + 1)}</span>);
-  }
+  const classes = useStyles();
+
+  const parts = React.useMemo(() => {
```

Comment:
Can you just add a comment about what this is doing?

Reply:
Done.

Reply:
Done.